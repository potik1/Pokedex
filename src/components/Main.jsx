import React, { Component } from 'react';
import Header from './Header';
import PokemonList from './PokemonList';
import PokemonDetail from './PokemonDetail';

class Main extends Component {
  render() {
    return (
      <div>
        <Header />
        <PokemonList />
      </div>
    );
  }
}

export default Main;
