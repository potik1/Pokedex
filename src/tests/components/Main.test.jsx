import React from 'react';
import { shallow } from 'enzyme';
import Main from '../../components/Main';

describe('Main', () => {
  const main = shallow(<Main />);

  test('renders correctly', () => {
    expect(main).toMatchSnapshot();
  });
});
