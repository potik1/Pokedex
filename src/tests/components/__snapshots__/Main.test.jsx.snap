// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <Main /> renders correctly 1`] = `
<div>
  <nav
    className="navbar navbar-light bg-faded"
  >
    <a
      className="navbar-brand"
      href="../"
    >
      POKEDEX
    </a>
  </nav>
  <div>
    Tu będą pokemony
  </div>
</div>
`;
